_posts/2022-03-14-ukraine-unterstuetzung.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Freifunk München hilft unbürokratisch Geflüchteten aus der Ukraine"
+date:   2022-03-14 13:00:00
+categories: freifunkmuc
+---
+
+Die aktuelle Welle von Flüchtlingen aus der Ukraine kommt auch in München an. Deshalb haben wir uns entschlossen, Hilfsorganisationen, Vereine und Unterkünfte unbürokratisch zu unterstützen, um den ankommenden Ukrainern eine stabile und gute Anbindung an das Internet zu ermöglichen, denn Internet ist in dieser Situation umso wichtiger und nicht nur ein "nice-to-have".
+
+### Einrichtung GOROD-Kulturzentrum
+
+Deswegen haben wir zusammen mit dem [GOROD Kulturzentrum](https://de.newgorod.org) auch sehr kurzfristig eine Flüchtlingsunterkunft mit Internet versorgt.
+Hier werden derzeit circa 800 Geräte mit derzeit circa 230 gleichzeitig verbundenen Clients mit circa 1 TB Traffic täglich versorgt. Anhand dessen sieht man deutlich, dass das Angebot gebraucht und genutzt wird.
+
+
+![Switch](/assets/posts/2022-03-14-gorod.png){: height="400px" }
+### Einrichtung AWO-Unterkunft Neuherbergstraße
+
+Auch die AWO-Unterkunft Neuherbergstraße hat mittlerweile von uns etwas Hardware spendiert bekommen. Aufgrund der fehlenden Breitbandanbindung vor Ort gehen wir in dem Fall sogar über LTE und versorgen zeitweise bis zu 120 Geräte gleichzeitig mit schnellem und zuverlässigem Internet.
+
+# Wie geht es weiter?
+
+Da soll jedoch nicht Schluss sein. Wir haben bereits neue Hardware bestellt, in der Hoffnung vielen weiteren Menschen zumindest eine Grundversorgung mit Internet anbieten zu können. Falls Ihr also Kontakte habt oder selber Hilfe bei der Versorgung von Unterkünften braucht, dann kontaktiert uns einfach. Wir helfen gerne!
+
+
+### Kontaktmöglichkeiten
+
+E-Mail: hilfe@ffmuc.bayern
+Twitter: https://twitter.com/FreifunkMUC
+Facebook: https://www.facebook.com/FreifunkMUC
+Chat: https://chat.ffmuc.net
```

Comment:
maybe direct link to Channel:
`[https://chat.ffmuc.net](https://chat.ffmuc.net/freifunk/channels/ukraine-hilfe---2022)`

Reply:
Added this to the "Wie geht es weiter?" section